src/gmean.js

```diff
@@ -0,0 +1,34 @@
+export default function gmean(values, valueof) {
+	const r = 64;
```

Comment:
Using tabs is inconsistent with the rest of the codebase, which uses spaces.

Reply:
I changed this commit a few times before pushing and didn't notice the last change was using tabs instead of spaces. Making the necessary changes..

> Using tabs is inconsistent with the rest of the codebase, which uses spaces.

Reply:
> I notice that tabs are used in the new file, whereas the rest of the codebase uses spaces.
> 
> Also curious, is this implementation derived from any existing work, or inspired from any particular source?

Yes, this PR was inspired by d3.mean(), used to calculate the arithmetic mean because I noticed similarities between the two. Will also work on showing differences between the geometric and arithmetic mean calculations and update this PR.